Codes_GitBash_So_Far.md

```diff
@@ -0,0 +1,86 @@
+Steps done until now in GitBash and GitHub
```

Comment:
If you use a markdown file, you need to add a space after # if you want to display a headline

Reply:
see [Cheatsheet for markdown](https://github.com/htw-pv3/literature/blob/master/Cheatsheet_MD.mdl)